packages/medulas-react-components/src/components/forms/SelectFieldForm/index.tsx

```diff
@@ -83,6 +83,7 @@ const SelectFieldForm = ({
         onChange(initial);
       }
     } catch (err) {}
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This is something that I need to debug and refactor, do not silent it.

Reply:
Could you instead create a ticket for that such that you have it in the tasks list? Without silencing it, the CI would fail.

Reply:
SGTM #143